SampleMerchant/public/index.html

```diff
@@ -53,19 +55,12 @@ <h1>Sample Merchant</h1>
           }
         };
 
-        function buildPaymentRequest() {
-          if (!window.PaymentRequest) {
-            return null;
-          }
-          const supportedInstruments = [
-            { supportedMethods: 'https://sample-pay-e6bb3.firebaseapp.com' }
-          ];
-          const details = {
+        var details = {
```

Comment:
`details` doesn't appear to be reassigned, should be `const` instead of `var`.

Reply:
done